main.go

```diff
@@ -16,10 +16,10 @@ func main() {
 			Name:    "uw-kafka-config",
 			Version: version,
 			Rules: []tflint.Rule{
-				rules.NewMskModuleBackendRule(),
-				&rules.MskAppTopics{},
-				&rules.MskTopicNameRule{},
-				&rules.MskTopicConfigRule{},
+				&rules.MSKModuleBackendRule{},
+				&rules.MSKAppTopics{},
```

Comment:
> Since you're touching on consistency, I've been thinking to rename this to MSKAppTopicsRule. WDYT?

[02812d6](https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/pull/15/commits/02812d63b8e9d8f143742c1c6b750e19dfdfa212)